### DISCUSSION TITLE
Multipart issues.

### DISCUSSION THREAD

**Discussion by cbertin1932** [actix/actix-web#3082] [General]
I am trying to get multipart to work properly, but having issues. For some reason, this function head, despite following the examples, will not let me go in the file.

```rust
#[post("/filesUpload")]
async fn files_upload(MultipartForm(form): MultipartForm<UploadData>,) -> Result<impl Responder, Error>{
```

But this works fine

```rust
#[post("/filesUpload")]
async fn files_upload(mut form: Multipart) -> Result<impl Responder, Error>{
```

Throught the middleware logger,  it is registering,, but no breakpoints are triggered nor log::info! called. My vue front-end will only give me an error when I close the actix server. cors::permissive for testing. Weird but I have an alternative, right? Well again in the docs this should be valid, right?

```rust
while let Some(item) = form.next().await {
   let mut field = item?;
   log::info!("{:?}",field.content_type);
}
```
Well the next() has red squiggles? Why? What am I doing wrong.

**Comment by robjtede** [comment]
> will not let me go in the file

I don't understand what you mean by this?

> Well the next() has red squiggles

this `next()` comes from `futures_util::StreamExt` trait, probably not in scope.
